packages/manager/apps/dedicated/package.json

```
@@ -35,6 +35,7 @@
     "@ovh-ux/manager-exchange": "^1.0.0 || ^2.0.0",
     "@ovh-ux/manager-filters": "^0.1.0 || ^1.0.0",
     "@ovh-ux/manager-iplb": "^0.4.0 || ^1.0.0",
+    "@ovh-ux/manager-metrics": "^0.0.0 || ^0.1.0",
```

```suggestion
    "@ovh-ux/manager-metrics": "^0.0.0 || ^1.0.0",
```
We could probably make a breking change commit on the module creation